python/ray/tests/test_ray_init_2.py

```diff
@@ -376,6 +378,20 @@ def test_driver_node_ip_address_auto_configuration(monkeypatch, ray_start_cluste
             )
 
 
+@pytest.fixture
+def short_tmp_path():
+    path = tempfile.mkdtemp(dir="/tmp")
+    yield path
+    shutil.rmtree(path)
+
+
+def test_temp_dir_with_node_ip_address(ray_start_cluster, short_tmp_path):
```

Comment:
Should we also test the `init(_temp_dir=x)` path, by initing first, then adding a node and checking if that node has the same temp_dir?

Reply:
I think adding a node to the cluster thats' started by ray.init is kind of weird pattern (I've never seen anyone doing this), so it is probably okay.